scrowl-io/scrowl
# Player

### Short summary

Major refactor of the codebase to employ standards and working towards aligning with the [design](https://eebos.github.io/scrowl_mockup/).

Created a SCORM runtime package to service the interactions between a project and the LMS.

Created the Player that transforms a project manifest into viewable content.

Created a template service that can compile handlebar templates

---

### Test steps

1. `yarn install`
1. ` yarn start:authoring`
1. Go to editor section and click `Publish`

- The app should start as expected
- The app navigation should work as expected
- `Publish' should create a SCORM zip in the `src/main/services/publisher/dist`
